allan_variance.py

```diff
@@ -7,25 +7,25 @@
 
 
 def allan_variance(x, dt=1, min_cluster_size=1, min_cluster_count='auto',
-                   n_clusters=100):
-    """Compute Allan variance (AV).
-    
+                   n_clusters=100, input_mode="increment"):
```

Comment:
> This is a question of the universality of the code. And the parameter is the easiest way.

You can quickly achieve that by just passing `x * dt` when necessary. It's even less text to type. But in this case I agree that having this parameter makes sense to point out this difference in the interpretation of the input.

> Can you help? ;)

Sure, I will thing about this later.